agentwire/server.py

```
@@ -774,6 +775,36 @@ async def _handle_dashboard_message(self, ws: web.WebSocketResponse, data: dict)
+        while asyncio.get_event_loop().time() < deadline:
+            try:
+                proc = await asyncio.create_subprocess_exec(
+                    "tmux", "capture-pane", "-p", "-t", local_name,
+                    stdout=asyncio.subprocess.PIPE,
+                    stderr=asyncio.subprocess.DEVNULL,
+                )
+                stdout, _ = await proc.communicate()
+                if stdout and stdout.strip():
+                    return True
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)